mava/systems/jax/mappo/networks.py

```diff
@@ -119,7 +168,19 @@ def make_networks(
     critic_layer_sizes: Sequence[int] = (512, 512, 256),
     observation_network: Callable = utils.batch_concat,
 ) -> PPONetworks:
-    """TODO: Add description here."""
+    """Calls functions to make discrete or continuous network
```

Comment:
Add  Raises descriptions from these lines:
`raise NotImplementedError(
            "Continuous networks not implemented yet."
            + "See: https://github.com/deepmind/acme/blob/"
            + "master/acme/agents/jax/ppo/networks.py"
        )`

Reply:
Thanks @OmaymaMahjoub, it's been added.